api/helpers/ find-spectatable-games.js

```diff
@@ -0,0 +1,23 @@
+const dayjs = require('dayjs');
+
+module.exports = {
+  friendlyName: 'Find Spectatable Games',
+
+  description: 'Finds all the games that are available to spectate',
+
+  fn: async (_, exits) => {
+
+    const recentUpdateThreshhold = dayjs().subtract(10, 'minute').valueOf();
```

Comment:
I think 10 minutes feels too long, 5 might be better. If you think about it no one is sitting on a turn for more than a minute or two at a time, and at 10 minutes I'm afraid you're just going to constantly jump in to empty games (though we never really have that many).

No harm going full bore first and tightening this up as needed, I'll leave it to you since you play more often than I do.